.github/workflows/python-can-run.yml

```diff
@@ -2,6 +2,10 @@ name: Python can run
 
 # Controls when the workflow will run
 on: [push]
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
I personally prefer to have the checks run on each push within the branch and not only on the PR. This helps to spot errors earlier on instead of at the end during the PR process

Reply:
I agree. I was trying to fix why there's only one workflow running for this PR, but:
1. Not sure if changing the workflow files here makes any difference for this PR
2. Not sure it fixes the issue